src/EnergyPlus/UnitarySystem.cc

```diff
@@ -16581,6 +16581,25 @@ namespace UnitarySystems {
         return airNode;
     }
 
+    int
+    getIndex(EnergyPlusData &state, std::string const &UnitarySysName, DataZoneEquipment::ZoneEquipType zoneEquipType, bool &errFlag)
+    {
+
+        int EquipIndex = 0;
```

Comment:
The index should be initialized to -1 for 0-based array

Reply:
@rraustad Corrected.